test/unit/inbound/process-submitted-settlement.test.js

```diff
@@ -0,0 +1,137 @@
+const mockCommit = jest.fn()
+const mockRollback = jest.fn()
+const mockTransaction = {
+  commit: mockCommit,
+  rollback: mockRollback
+}
+
+jest.mock('../../../app/data', () => {
+  return {
+    sequelize:
+       {
+         transaction: jest.fn().mockImplementation(() => {
+           return { ...mockTransaction }
+         })
+       }
+  }
+})
+
+jest.mock('../../../app/inbound/get-settlement-by-invoicenumber-and-value')
+const getSettlementByInvoiceNumberAndValue = require('../../../app/inbound/get-settlement-by-invoicenumber-and-value')
+
+jest.mock('../../../app/inbound/save-settlement')
+const saveSettlement = require('../../../app/inbound/save-settlement')
+
+const processReturnSettlement = require('../../../app/inbound/process-return-settlement')
+const settlement = JSON.parse(JSON.stringify(require('../../mock-settlement')))
```

Comment:
line 26 should read `let settlement`

then on line 30 it should be `settlement = JSON.parse(JSON.stringify(require('../../mock-settlement')))`